### PR TITLE
[metasrv] refactor: rename test function names

### PR DESCRIPTION
**Pull request by drmingdrmer**
I hereby agree to the terms of the CLA available at: https://databend.rs/policies/cla/

## Changelog

- Improvement

## Related Issues

### CODE REVIEW

File: common/building/src/lib.rs [hunk 1]
```diff
@@ -45,7 +45,7 @@ pub fn add_env_vergen() {
 
 pub fn add_env_commit_authors() {
     let r = run_script::run_script!(
-        // use eamil to uniq authors
+        // use email to uniq authors
```
Comment (PsiACE) [inline common/building/src/lib.rs:48]: :+1:

**Review by PsiACE — APPROVED** [review]
LGTM

**Comment by BohuTANG** [comment]
/lgtm
Neat man!

**Comment by BohuTANG** [comment]
/lgtm
Neat man!

**Pull request merged**